lib/modules/subredditManager.js

```diff
@@ -777,6 +782,8 @@ function redrawSubredditBar() {
 				if (module.options.linkMod.value) shortCutsHTML += `<span class="separator">-</span><a class="subbarlink ${specialButtonSelected.mod}" href="/r/mod/">Mod</a>`;
 				if (module.options.linkModqueue.value) shortCutsHTML += '<span class="separator">-</span><a class="subbarlink" href="/r/mod/about/modqueue">Modqueue</a>';
 			}
+
+			if (module.options.linkSaved.value) shortCutsHTML += '<span class="separator">-</span><a class="subbarlink" href="/user/me/saved">Saved</a>';
```

Comment:
It is inside a loggedInUser() if-statement
Check previous lines to confirm this